examples/src/main/java/io/kubernetes/client/examples/InformerExample.java

```diff
@@ -11,11 +11,20 @@
 import io.kubernetes.client.models.V1ObjectMeta;
 import io.kubernetes.client.util.CallGeneratorParams;
 import io.kubernetes.client.util.Config;
+import java.util.concurrent.TimeUnit;
 
+/**
+ * A simple example of how to use the Java API
+ *
+ * <p>Easiest way to run this: mvn exec:java
+ * -Dexec.mainClass="io.kubernetes.client.examples.InformerExample"
+ *
+ * <p>From inside $REPO_DIR/examples
+ */
 public class InformerExample {
   public static void main(String[] args) throws Exception {
-
     ApiClient client = Config.defaultClient();
+    client.getHttpClient().setReadTimeout(0, TimeUnit.SECONDS); // infinite timeout
```

Comment:
apply the infinite timeout to the watch example

Reply:
done